### PR TITLE
Hide open in editor (simple version)

### PR DESCRIPTION
**Pull request by BenRussert**
This is much cleaner if we are not worried about the icon switching.

I am not crazy about the way the tooltip message looks, open to suggestions on that (or anything else)

simple-ctrl-click mp4

### CODE REVIEW

File: lib/components/result-view/result-view.js [hunk 1]
```diff
@@ -48,8 +56,9 @@ class ResultViewComponent extends React.Component {
     comp.add(
       atom.tooltips.add(element, {
         title: this.props.store.executionCount
-          ? `Copy to clipboard (Out[${this.props.store.executionCount}])`
-          : "Copy to clipboard"
+          ? `Click to copy, ctrl/cmd click to open in editor (Out[${this.props
```
Comment (BenRussert) [inline lib/components/result-view/result-view.js]: That's a great idea. I'll add that later today.

Comment (lgeiger) [inline lib/components/result-view/result-view.js]: Cool thanks!

**Comment by BenRussert** [comment]
@lgeiger I haven't actually tried this on mac, but pretty sure it will work.

**Comment by lgeiger** [comment]
> @lgeiger I haven't actually tried this on mac, but pretty sure it will work.

Works excellent! Thanks.

**Pull request merged**